### DISCUSSION TITLE
Welcome to RealDebridManager Discussions!

### DISCUSSION THREAD

**Discussion by hyperbunny77** [hyperbunny77/RealDebridManager#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by bhullIT** [comment]
Thanks for this!
Is Alldebrid supported?
If not, are there any plans or ETA?
